src/cli.ts

```diff
@@ -172,14 +172,18 @@ prog.command('build [dest]')
 			await _build(opts.bundler, opts.legacy, opts.cwd, opts.src, opts.routes, opts.output, dest, opts.ext);
 
 			const launcher = path.resolve(dest, 'index.js');
+			const pkg = await import(path.resolve(opts.cwd, 'package.json'));
```

Comment:
We should probably just use `JSON.parse(fs.readFileSync(...))` or something similar here for this. Using a dynamic `import()` is confusing because ESM in modern version of Node doesn't actually work with JSON files currently. This only succeeds because this code is getting transpiled into

```js
const pkg = yield Promise.resolve().then(function () { return require(path.resolve(opts.cwd, 'package.json')); });
```